inst/WORDLIST

```diff
@@ -55,185 +69,196 @@ Durations
 EMA
 EOSSTT
 FACM
+FCVD
+Fibrinogen
 Findability
 Framingham
 Fridericia
 Fridericia's
 Fujimoto
+GGT
 GSK
-GUILLAIN
 Gehan
-Github
 GlaxoSmithKline
```

Comment:
Sort of related but didn't @fshanlee mention something about an official name change? Should we make it a separate issue for the release?

Reply:
Oh yea - we research that and our legal department says we are fine to leave as is